packages/cli/src/commands/org/search/dump.ts

```diff
@@ -279,19 +276,18 @@ export default class Dump extends Command {
     }) as SingleBar;
   }
 
-  private async aggregateResults(newResults: RawResult[]) {
+  private async aggregateResults(results: RawResult[]) {
     const maxAggregatedResults = (await this.parse(Dump)).flags.chunkSize;
-    const amountOfNewResultsToExtract =
-      maxAggregatedResults - this.aggregatedResults.length;
-    this.aggregatedResults.push(
-      ...newResults.splice(
-        0,
-        Math.min(amountOfNewResultsToExtract, newResults.length)
-      )
-    );
-    if (newResults.length > 0) {
+    while (results.length > 0) {
```

Comment:
We can refactor like this because the splice was already noop if `results.length===0` :
1. `Math.min(amountOfNewResultsToExtract, results.length) == 0`
2. `[].splice(0,0) == []`
3. array.push(...[]) does nothing


However, that makes the method dumping the result no matter what in the end.
This was first made by the other chunk that was removed above.
I checked the order of operations and I don't think that it would change anything.